requirements-sdp.txt

```diff
@@ -14,3 +14,75 @@
 #     conda activate sdp
 #     pip install -e .[test]
 #     pip freeze | grep -v jwst.git >> requirements-sdp.txt
+asdf==2.11.1
+asdf-astropy==0.2.1
+asdf-coordinates-schemas==0.1.0
+asdf-standard==1.0.2
+asdf-transform-schemas==0.2.2
+asdf-wcs-schemas==0.1.1
+astropy==5.0.4
+attrs==21.4.0
+BayesicFitting==3.0.1
+certifi==2021.10.8
+charset-normalizer==2.0.12
+ci-watson==0.6.1
+codecov==2.1.12
+colorama==0.4.4
+coverage==6.3.2
+crds==11.13.1
+cycler==0.11.0
+drizzle==1.13.4
+filelock==3.6.0
+flake8==4.0.1
+fonttools==4.33.3
+future==0.18.2
+getch==1.0
+gwcs==0.18.1
+idna==3.3
+imageio==2.19.0
+iniconfig==1.1.1
+jmespath==1.0.0
+jsonschema==4.4.0
```

Comment:
setup.cfg is at 4.0.1

Reply:
The `setup.cfg` defines them as `>=` . It will work with the older versions. The sdp is fixed to whatever is available at the time of installation. Tests are also running with these versions so it should be fine. 

```
py310 installed:  ... jsonschema==4.4.0 ...
```